.codecov.yml

```diff
@@ -1,6 +1,23 @@
 comment: false
 coverage:
   status:
+    # Patch status gives meaningless results whenever a test is modified without changing the
+    # correspondingly tested code, and may be difficult to interpret without context. Disable
+    # posting it to GitHub.
+    patch: false
     project:
-      default:
-        threshold: 1
```

Comment:
You might keep the threshold option.  Our javascript coverage in particular tends to vary from run to run due to async timing.

Reply:
This is obsoleted by the `target` option. The check now passes if the coverage is above the threshold (which means we can insist on it always passing).